docs/manage/command_line.rst

```diff
@@ -141,66 +141,51 @@ You will be prompted to input the **Username** and **Password** and the new **su
 The full name for the new super admin user will be the same as the chosen ``username``, and can be edited in the **Facility > Users** page, or the user profile.
 
 
-Import Users from a CSV File
-****************************
+Import and Export User Data from a CSV File
```

Comment:
one day I'd love if we switched our headers to sentence case...

### Import and export user data from a CSV file

Reply:
Me too, and I plan to address that refactor once we have the documentation style guide in place (read as _with this year's GSoD work_ :wink:)